source/Tools/src/GitRevision.cc

```diff
@@ -1,4 +1,4 @@
 #include <GitRevision.h>
 
 //GitRevision.cc - written by cmake. changes will be lost!
-const std::string GitRevision::revisionNumber = "devLite-798 on git@github.com:drasal/tkLayout.git";
+const std::string GitRevision::revisionNumber = "devLite-900 on git@github.com:ghugo83/tkLayout.git";
```

Comment:
Can you, please, do git checkout  source/Tools/src/GitRevision.cc? GitRevision.cc change is artificial... This is something, what comes out of CMake compilation, if one removes content of build directory without doing make uninstall. That's the weak part of CMake compilation...